_includes/_components/domain-event.html

```diff
@@ -1,50 +1,133 @@
-<!-- TODO -->
-
-      <article class="component dark-blue">
-        <header>
-          <div class="container">
+<article class="component dark-blue">
+    <header>
+        <div class="container">
             <h2>Domain Event</h2>
-            <p>This component blablabla</p>
-          </div>
-        </header>
-        <div class="container content">
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-          </p>
-          <div class="row">
+            <p>The raise method allows you to trigger any event on your entity and dispatched this Knp\Rad\DomainEvent\Event once the entity has been flushed.</p>
```

Comment:
`The raise method allows you to trigger any event in your entity and a Knp\Rad\DomainEvent\Event once the entity is flushed.`